api/v2/utilities/validators.py

```diff
@@ -0,0 +1,44 @@
+import re
+from flask import request, jsonify, make_response
+from functools import wraps
+
+
+def validate_bouquet_adding(func):
+    """Decorator to validate the addition of a bouquet"""
+
+    @wraps(func)
+    def wrapper():
+        if func.__name__ == 'post':
+            request_data = request.get_json()
+
+            def check_numeric(_value):
+                _re = bool(re.fullmatch(r'^[a-zA-Z]*\s?[a-zA-Z]*\s?[a-zA-Z]*', str(value)))
+                return _re
+
+            def check_url(url):
+                _re = bool(re.fullmatch(
+                    r'(https?:\/\/(?:www\.|(?!www))[a-zA-Z0-9][a-zA-Z0-9-]+[a-zA-Z0-9]\.[^\s]{2,}|'
+                    r'www\.[a-zA-Z0-9][a-zA-Z0-9-]+[a-zA-Z0-9]\.[^\s]{2,}|'
+                    r'https?:\/\/(?:www\.|(?!www))[a-zA-Z0-9]+\.[^\s]{2,}|'
+                    r'www\.[a-zA-Z0-9]+\.[^\s]{2,})', url))
+                return _re
+
+            try:
```

Comment:
You should consider flask restful's `reqparse` to handle the key validation you are doing here. It is a lot more cleaner. Check out this link -> `https://flask-restful.readthedocs.io/en/0.3.5/reqparse.html